### PR TITLE
Add support for Chromium driver

### PR DESCRIPTION
**Pull request by mnocon**
JIRA: https://jira.ez.no/browse/EZP-30669

Adding support for Chromium driver in our Docker stack.
Related PR: https://github.com/ezsystems/ezplatform/pull/417

### CODE REVIEW

**Comment by mnocon** [comment]
Travis failure is unrelated, but I'll have a look at it

**Review by micszo — APPROVED** [review]

File: php/scripts/docker-entrypoint.sh [hunk 1]
```diff
@@ -28,7 +28,10 @@ if [ "$BEHAT_SELENIUM_HOST" != "" ] && [ "$BEHAT_WEB_HOST" != "" ]; then
     logger "Copying behat.yml.dist to behat.yml and updating selenium and web hosts"
     if [ -f behat.yml.dist ]; then
         cp -f behat.yml.dist behat.yml
+        sed -i "s@javascript_session: selenium@javascript_session: ${MINK_DEFAULT_SESSION}@" behat.yml
         sed -i "s@localhost:4444@${BEHAT_SELENIUM_HOST}:4444@" behat.yml
+        sed -i "s@localhost:4444@${BEHAT_SELENIUM_HOST}:4444@" behat.yml
+        sed -i "s@localhost:9222@${BEHAT_CHROMIUM_HOST}:9222@" behat.yml
```
Comment (andrerom) [inline php/scripts/docker-entrypoint.sh]: Maybe have a default fallback on these new envs? (MINK_DEFAULT_SESSION and BEHAT_CHROMIUM_HOST)

Comment (mnocon) [reply php/scripts/docker-entrypoint.sh]: You're right - I was thinking that they will be defined in metarepo (added in corresponding PR to .env), but they don't have to be - I'll take that into account soon

Comment (mnocon) [reply php/scripts/docker-entrypoint.sh]: Addressed in https://github.com/ezsystems/docker-php/pull/39/commits/80bd5f09ac052191342ce5a529c8516295708b6d - Ive decided not to set any default values here, as they should be already present in behat.yml.dist

**Review by m-tyrala — APPROVED** [review]

File: php/scripts/docker-entrypoint.sh [hunk 2]
```diff
@@ -28,7 +28,10 @@ if [ "$BEHAT_SELENIUM_HOST" != "" ] && [ "$BEHAT_WEB_HOST" != "" ]; then
     logger "Copying behat.yml.dist to behat.yml and updating selenium and web hosts"
     if [ -f behat.yml.dist ]; then
         cp -f behat.yml.dist behat.yml
+        if [ "$MINK_DEFAULT_SESSION" != "" ]; then sed -i "s@javascript_session: selenium@javascript_session: ${MINK_DEFAULT_SESSION}@" behat.yml; fi
         sed -i "s@localhost:4444@${BEHAT_SELENIUM_HOST}:4444@" behat.yml
+        sed -i "s@localhost:4444@${BEHAT_SELENIUM_HOST}:4444@" behat.yml
```
Comment (andrerom) [inline php/scripts/docker-entrypoint.sh]: is this line needed? seems to be duplicate of above line

Comment (mnocon) [reply php/scripts/docker-entrypoint.sh]: Thanks! Removed 😓

**Review by andrerom — APPROVED** [review]

**Comment by andrerom** [comment]
hmm, but seems tests are broken by the addition of yarn in 2.5.

Is it this part we need to adjust to use latest-node?
https://github.com/ezsystems/docker-php/blob/c6efa212aa643151c2fe4181543c13d3604be1e7/bin/.travis/test.sh#L47-L53

**Comment by andrerom** [comment]
As for PHP 5.6 - 7.0 it seems to be issue with packages for Debian jessie, but given 7.1 is also jessie based it might just be temporary glitch:
```
W: Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/InRelease  Unable to find expected entry 'main/binary-amd64/Packages' in Release file (Wrong sources.list entry or malformed file) 
```

7.2 failure seems to be temprary, bad gateway trying to download blackfire.

**Comment by mnocon** [comment]
About the PHP5.6 images - seems like they've been dropped from https://hub.docker.com/_/php (https://github.com/docker-library/php/commit/e9320fdb752edb2fb5d1be8412172f5c78255a45)

**Comment by mnocon** [comment]
The part you linked to looks like a good place to fix 7.x builds, I'll commit a change there to verify

**Comment by andrerom** [comment]
> About the PHP5.6 images - seems like they've been dropped from https://hub.docker.com/_/php (docker-library/php@e9320fd)

They have stopped building new as it is no longer new releases coming for 5.6, but they existing tags still exists. We can drop it in separate PR, but the issue here is not 5.6 afaik.

**Comment by andrerom** [comment]
@mnocon for the failures, see here: https://github.com/ezsystems/docker-php/pull/38

**Comment by mnocon** [comment]
@andrerom thanks, I'll rebase this PR after that one is merged

**Comment by mnocon** [comment]
@andrerom please have a look again - I've rebased with latest master.

The build for PHP5.6 was still red, because today new version of redis has become available on PECL, without PHP5.6 support - I'm forcing the install of latest stable that supports PHP5.6 (4.3.0) instead. Versions: https://pecl.php.net/package/redis

Please also have a look at https://github.com/ezsystems/ezplatform/pull/417/files , another PR that uses variables introduced in this one.

**Review by andrerom — APPROVED** [review]

**Review by vidarl — APPROVED** [review]

**Pull request merged**